src/components/elementView/detailedView/propertiesView.scss

```diff
@@ -1,5 +1,6 @@
 .propsview .row-props {
   display: flex;
+  white-space: pre-line;
```

Comment:
important change!! had to add this so that propertiewView renders new lines. If you do a refactor of propertiesview,  this should def still be included so we can see new lines rendered correctly